tools/detect-version-up-of-databases.rb

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env ruby
+
+require 'open-uri'
+require 'nokogiri/class_resolver'
+require 'nokogiri'
+
+target_database_names = [
+  "mysql-5.7",
+  "mysql-8.0",
+#  "mariadb-10.11",
+#  "mariadb-10.10",
+#  "mariadb-10.9",
+#  "mariadb-10.8",
+#  "mariadb-10.7",
+#  "mariadb-10.6",
+#  "mariadb-10.5",
+#  "mariadb-10.4",
+#  "percona-server-5.7",
+#  "percona-server-8.0",
+]
+
+if ARGV.size < 1
+  puts("Usage: #{$0} TARGET_DATABASE_NAME")
+  puts("Target databases list:: ")
+  target_database_names.each do |target_database_name|
+    puts(target_database_name)
+  end
+  exit(false)
+end
+
+
+def local_latest_version_mysql(target)
+  target_package_name = "mysql-community-#{target}-mroonga"
+  File.open("#{__dir__}/../packages/#{target_package_name}/yum/#{target_package_name}.spec.in",
+            "r") do |file|
```

Comment:
* `target_` prefix is redundant in this context
* `"r"` is redundant.

```suggestion
  package_name = "mysql-community-#{target}-mroonga"
  spec_in_path = "#{__dir__}/../packages/#{target_package_name}/yum/#{target_package_name}.spec.in"
  File.open(spec_in_path) do |spec_in|
```